src/com/carolinarollergirls/scoreboard/defaults/DefaultScoreBoardModel.java

```diff
@@ -351,7 +326,9 @@ private void _endTimeout() {
 		if (!tc.isRunning()) { return; }
 		
 		requestBatchStart();
-		tc.stop();
+		if (!frontendSettings.get(FRONTEND_SETTING_CLOCK_AFTER_TIMEOUT).equals(Clock.ID_TIMEOUT)) {
```

Comment:
Having frontend settings affect clocks like this doesn't feel right. Is there a way we could do this consistently in the backend, and have the fronted do the main logic?

Having it this way also means if it's set that timeout clock is running, then there's no feedback/encouragement for the operator to actually end the timeout - which won't get us towards consistent practices.

Reply:
I understand the feeling. But the only other option I see would be to have both clocks running in the backend and madify all the frontends (including ones, we don't maintain) to choose the clock for display based on this setting. That's a lot of work just to please some loud people who don't understend "the end of a timeout is marked by a long rolling whistle".

And given that the requested functionality explicitly is to not change the displayed clock, I'm not sure how to give more feedback beyond unsetting timeout type.